app/components/views/FatalErrorPage.js

```diff
@@ -38,7 +38,15 @@ class FatalErrorPage extends React.Component {
         </>);
       break;
     default:
-      errorMessage = <T id="fatal.suggestion.fallthrough" m="Please note the error above and go to the support channel on slack/matrix/rockchat for help resolving the issue." />;
+      errorMessage = (
+        <T id="fatal.suggestion.fallthrough"
+          m="Please note the error above and go to the support channel on slack/matrix/Rocketchat for help resolving the issue. {link}"
```

Comment:
I'd remove slack and rocket chat since we give preference to matrix now.

Reply:
I agree, so I added the following message: `Please note the error above and go to the support channel on matrix (via riot) or some other preferred chat channel for help resolving the issue.`

Reply:
"(via riot)" is unnecessary. Users can choose whatever client they want.

Reply:
Removed